src/main/java/io/javalin/core/JavalinServer.kt

```diff
@@ -27,6 +27,7 @@ import javax.servlet.http.HttpServletResponse
 class JavalinServer(val config: JavalinConfig) {
 
     var serverPort = 7000
+    var serverHost = "localhost"
```

Comment:
The docs state 

```
If null or 0.0.0.0, then bind to all interfaces.
```
So I guess we should have either `null` or `0.0.0.0` as the default. Maybe `0.0.0.0` with a comment?